client/src/components/common/Tutorial.tsx

```diff
@@ -0,0 +1,210 @@
+import React, {useCallback, useEffect, useState} from "react";
+import styled from "@emotion/styled";
+
+import {MENU_HEIGHT, TUTORIAL_CONTENTS, TUTORIAL_TITLE} from "../../domains/constants";
+
+const TutorialLayout = styled.div`
+  > div.dropback {
+    display: block;
+    position: absolute;
+    z-index: 4;
+    opacity: 0.8;
+    background-color: rgba(0,0,0,0.6);
+  }
+ 
+  > div.top {
+    top: 0;
+    height: ${MENU_HEIGHT}px;
+    width: 100%;
+  }
+  
+  > div.left {
+    top: ${MENU_HEIGHT}px;
+    left: 0;
+  }
+  
+  > div.right {
+    right: 0;
+    height: 100%;
+  }
+`;
```

Comment:
`.top`, `.left`, `.right` 같은 경우는 항상 `.dropback`과 함께 쓰일 거라면 nested된 구조로 가도 괜찮겠네요.

```suggestion
const TutorialLayout = styled.div`
  > div.dropback {
    display: block;
    position: absolute;
    z-index: 4;
    opacity: 0.8;
    background-color: rgba(0,0,0,0.6);
    
    &.top {
      top: 0;
      height: ${MENU_HEIGHT}px;
      width: 100%;
    }
  }
  ...
```